phpBB/phpbb/cache/driver/redis.php

```diff
@@ -137,11 +141,7 @@ function _read($var)
 	*/
 	function _write($var, $data, $ttl = 2592000)
 	{
-		if ($ttl == 0)
-		{
-			return $this->redis->set($var, $data);
-		}
-		return $this->redis->setex($var, $ttl, $data);
+		return $this->redis->set($var, $data, ['EXAT' => time() + $ttl]);
```

Comment:
EX should be the same as the previous TTL?

Reply:
Result is the same, I just made it consistent with other phpBB cache drivers which set timestamp expiration point rather than relative expiration timespan.